myhpom/forms/profile.py

```diff
@@ -30,6 +30,21 @@ class Meta:
         widget=forms.TextInput(attrs={'class': 'form-control'}),
     )
 
+    def clean(self):
+        cleaned_data = super(EditUserForm, self).clean()
+        
+        # check to ensure that the email is available
+        if User.objects.filter(email=cleaned_data.get('email')).exclude(id=self.instance.id).exists():
+            self.add_error('email', "Email already in use.")
+
+        if not self.is_valid():
+            self.errors['__all__'] = [
+                'Please make the indicated corrections to edit your profile.'
```

Comment:
The language is a little confusing here - maybe just `Please make the following corrections`? Also...when I test this manually the message never appears - it looks like there is no `non_field_errors' in the template.

Reply:
I ended up not adding the non_field_errors to the template, since we haven't had it before, but I agree it makes sense to have it, so ... adding!